camp/camp-brooklyn/src/test/java/org/apache/brooklyn/camp/brooklyn/WorkflowYamlTest.java

```diff
@@ -798,6 +800,41 @@ public void testLockReleasedOnCancel() throws Exception {
         Asserts.assertEquals(x2.getTask(false).get().getUnchecked(Duration.seconds(5)), "done");
     }
 
+    // prior to executing this test make sure you create an ec2 instance and tag it with "a2/vm"
```

Comment:
Is it worth writing a simpler test that just does echo where the args are string including inline environment variables?

Reply:
We wrote simple echo tests and this is how we missed the fact we did not support command containers. :)

But yeah, I see what you mean. I will add an example using the terraform container just to query the version. Thx!